site/content/docs/custom-theme.md

```diff
@@ -0,0 +1,138 @@
+---
+title: How to theme your Flowershow website
+description: Learn how to add custom fonts and colors to your site
+type: Blog
+created: 2023-01-23
+---
+
+We are going to learn how to change the fonts and colors used throughout
+your Flowershow website, so you can have it looking exactly the way you want!
+
+We will be changing the default site:
+
+![[defaultfont.png]]
+
+To this:
+
+![[lightafter.png]]
+
+Flowershow uses [Tailwind CSS](https://tailwindcss.com/) to style its websites, thus customization of fonts and colors is going to be done via tailwindcss through the use of its config file.
+
+The config file is called `tailwind.config.js` and located in the directory where your Flowershow app is installed, usually a `.flowershow` subfolder wherever you ran `npx flowershow install`.
+
+> If you want to change the theme of only one section, then that should be done using HTML/Tailwindcss inside that specific markdown file
+
+## How to customize the fonts of your Flowershow website
+
+We are going to change the heading font to _Kanit_ and body font to _Roboto_ across the whole site by following these steps:
+
+1. Import _Kanit_ and _Roboto_ fonts from https://fonts.google.com/
+
+2. Paste what you copied at the top of the `global.css` file, which can be found in your flowershow directory's style folder - `.flowershow/site/styles/global.css`
+
+```css=
+@import url('https://fonts.googleapis.com/css2?family=Kanit:wght@900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+```
+
+3. By default, Tailwind provides three font family utilities: a cross-browser sans-serif stack, a cross-browser serif stack, and a cross-browser monospaced stack. You can change, add, or remove these by editing the theme.fontFamily section of your Tailwind config. Open the file `tailwind.config.js` in `.flowershow` and add the new fonts by extending the fontFamily property of `theme` as shown below.
+
+Change the **default config**
+
+```js=
```

Comment:
I don't think it's necessary to repeat code here (from - to). I would just say change the sans and heading values to your imported fonts' name as seen in the code snippet below. 

Also mention what sans is used for eg. changes the body's font and headings for `# Your heading` but maybe that's self explanatory.